spec/scripts/graphql_api_content/render_helpers_spec.rb

```diff
@@ -703,9 +708,7 @@
             <h1 class="has-pills" data-algolia-exclude>__DirectiveLocation<span class="pill pill--object pill--normal-case pill--large"><code>OBJECT</code></span></h1>
             <!-- vale on -->
```

Comment:
cool work with frontmatter 🤩 

In this case we can remove the `<!-- vale on/off -->` comments in the GraphQL API render script

Reply:
Had a quick look at doing this but it appears we're still making use of this to get around some vale violations from